#!/bin/sh /usr/share/dpatch/dpatch-run
## 341-magic-add-bzr.dpatch by Jelmer Vernooij <jelmer@samba.org>
##
## DP: Adds new magics for bzr (Closes: #488742).

@DPATCH@

diff -Naurp file.orig/magic/Magdir/revision file/magic/Magdir/revision
--- file.orig/magic/Magdir/revision	2008-07-01 15:42:16.000000000 +0200
+++ file/magic/Magdir/revision	2008-07-01 15:43:34.000000000 +0200
@@ -22,3 +22,9 @@
 # From:	Uwe Zeisberger <zeisberg@informatik.uni-freiburg.de>
 0	string	SVN-fs-dump-format-version:	Subversion dumpfile
 >28	string	>\0				(version: %s)
+
+# Type:	Bazaar revision bundles and merge requests
+# URL:	http://www.bazaar-vcs.org/
+# From:	Jelmer Vernooij <jelmer@samba.org>
+0	string	#\ Bazaar\ revision\ bundle\ v Bazaar Bundle
+0	string	#\ Bazaar\ merge\ directive\ format Bazaar merge directive
